source code does not compile

... on the stable Debian GNU/Linux release, following the INSTALL.md's `cmake ..; make`:

```
In file included from /home/user/CombiFF/src/shared/Range.cpp:3:
/home/user/CombiFF/src/shared/Range.h:25:33: error: ‘string’ is not a member of ‘std’
   25 | typedef std::unordered_map<std::string, RangeRestriction> RangeMap;
      |                                 ^~~~~~
/home/user/CombiFF/src/shared/Range.h:10:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
    9 | #include <vector>
  +++ |+#include <string>
   10 | 
/home/user/CombiFF/src/shared/Range.h:25:33: error: ‘string’ is not a member of ‘std’
   25 | typedef std::unordered_map<std::string, RangeRestriction> RangeMap;
      |                                 ^~~~~~
/home/user/CombiFF/src/shared/Range.h:25:33: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
/home/user/CombiFF/src/shared/Range.h:25:57: error: template argument 1 is invalid
   25 | typedef std::unordered_map<std::string, RangeRestriction> RangeMap;
      |                                                         ^
/home/user/CombiFF/src/shared/Range.h:25:57: error: template argument 3 is invalid
/home/user/CombiFF/src/shared/Range.h:25:57: error: template argument 4 is invalid
/home/user/CombiFF/src/shared/Range.h:25:57: error: template argument 5 is invalid
/home/user/CombiFF/src/shared/Range.h:27:23: error: scalar object ‘combi_ff::possible_ranges’ requires one element in initializer
   27 | static const RangeMap possible_ranges{
      |                       ^~~~~~~~~~~~~~~
make[2]: *** [src/shared/CMakeFiles/sharedLib.dir/build.make:134: src/shared/CMakeFiles/sharedLib.dir/Range.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:254: src/shared/CMakeFiles/sharedLib.dir/all] Error 2
make: *** [Makefile:103: all] Error 2
```

Cmake output looks normal:

```
$ cmake ..
-- The C compiler identification is GNU 10.2.1
-- The CXX compiler identification is GNU 10.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- No build type selected, default to Release. ** If you want to use debug mode, use -DCMAKE_BUILD_TYPE="Debug" **
-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/CombiFF/build
```

--- a/src/shared/Range.h
+++ b/src/shared/Range.h
@@ -7,6 +7,7 @@
 #include <unordered_map>
 #include <utility>
 #include <vector>
+#include <string>
 
 namespace combi_ff {
 

